src/utils/withdrawals.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Gets the withdrawal UUID specific to the associated vault.
+ * @param uuid The withdrawal UUID
+ * @returns The withdrawal uuid specific to the associated vault.
+ */
+export const getWithdrawalVaultUUID = (uuid: bigint) =>
```

Comment:
This is only used in `src/actions/public/simple-bitcoin-vault-state.ts` and `getVaultUUID` is not even used. I think this function could be moved to that file and this file removed to keep things simple/avoid over-engineering. Unless we know we will use `getVaultUUID` in some other place soon.

Anyway, having a "utils" file seems to break the structure of this extension. 🤔

Reply:
> Unless we know we will use getVaultUUID in some other place soon.

Not so far, but as it took a few attempts to get the bit operations right, I wanted to preserve the implementation 😅  even though it is not used.

But. you're correct, it's not that useful. I will remove it

Reply:
updated in [303a825](https://github.com/hemilabs/hemi-viem/pull/33/commits/303a825d4447ad05a89629a89c84e62b5c06a8cd)

Reply:
I also had to remove the tests [529337e](https://github.com/hemilabs/hemi-viem/pull/33/commits/529337e28b81c741a2d41804ce13c75a55b437f8) as the function can't be in the same file where it is used, as all exported functions are exposed through the decorators objects...

Only way to test it would be to mock the contracts call, but it seems to me a lot of effort for just one test. So I removed them, as well as `vitest`